### ISSUE TITLE
integrate com_favorites

### ISSUE THREAD

**Issue by ChrisFrench** [dioscouri/tienda#7]

**Comment by rdiaztushman** [comment]
Moving to a new-features milestone.  Is this already in progress, though?

**Comment by ChrisFrench** [comment]
Nope I haven't started it but should be easy. I will get to it soon

**Comment by rdiaztushman** [comment]
Ok, then let's leave it on hold for a bit.  It's more important that we get TIenda working with few bugs before we start adding new features
